### PR TITLE
DDF-2824 Clean up Search UI terminology

### PR DESCRIPTION
**Pull request by brendan-hofmann**
#### What does this PR do?
Adjust the name of the Standard Search UI App to more accurately reflect the features it contains.
#### Who is reviewing it? 
(please choose AT LEAST two reviewers that need to approve the PR before it can get merged)
@vinamartin 
@djblue 
#### Select relevant component teams: 
https://github.com/orgs/codice/teams
#### Ask 2 committers to review/merge the PR and tag them here. If you don't know who to ask, you can request reviews in https://groups.google.com/forum/#!forum/ddf-developers .
(please choose ONLY two committers from below, delete the rest)
@andrewkfiedler
@tbatie
#### What are the relevant tickets?
[](https://codice.atlassian.net/browse/)
#### Checklist:
- [x] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: catalog/spatial/geowebcache/README.md [hunk 1]
```diff
@@ -88,7 +88,7 @@ diskquota.xml:
 See http://geowebcache.org/docs/current/configuration/diskquotas.html for more information on configuration options for disk quota.
 
 ### Configure the Standard Search UI
```
Comment (vinamartin) [inline catalog/spatial/geowebcache/README.md]: ❓ Should this be changed?

Comment (brendan-hofmann) [reply catalog/spatial/geowebcache/README.md]: Good catch. 👍

**Comment by brendan-hofmann** [comment]
build now

**Comment by brendan-hofmann** [comment]
build now

**Comment by brendan-hofmann** [comment]
build now

**Comment by brendan-hofmann** [comment]
Network error. build now

**Comment by brendan-hofmann** [comment]
build now

**Comment by brendan-hofmann** [comment]
Network error. build now

**Pull request merged**